internal/util/sessionutil/session_util.go

```diff
@@ -301,6 +302,14 @@ func (s *Session) getCompleteKey() string {
 	return path.Join(s.metaRoot, DefaultServiceRoot, key)
 }
 
+func (s *Session) getSessionKey() string {
```

Comment:
Already have method `getCompleteKey` with support of active-standby, can use it directly. If enable Active-standby, should watch xxxcoord-ID instead of xxxcoord. Standby coord will also watch the delete event and quit when Active coord disappear.

Reply:
no, we should watch the active key `xxxcoord` without prefix. When active key 'xxxcoord' is deleted, the active component should be kill. Then standby component upgrade to an active component.
If component is a node, the active key is `xxxnode-ID`.

Reply:
We need to ensure that only one component is active anytime.